### DISCUSSION TITLE
Why my contribution does not show up?

### DISCUSSION THREAD

**Discussion by Winbobob** [community/community#78044] [Other Feature Feedback, Questions, & Ideas] [answered]
I've made [more than a dozen contributions (pull requests merged to the main branch)](https://github.com/envoyproxy/envoy/commits?author=Winbobob) to Envoy proxy repo, but I cannot find myself here: https://github.com/envoyproxy/envoy/graphs/contributors?from=2022-12-28&to=2023-12-06&type=c. Any ideas?

Btw, I've add the email that made the contribution to my GitHub account.

**Comment by OshekharO** [comment, marked as answer]
Github shows only the top 100 contributors for a project.

If there are more than 100, the other contributors beyond these top 100 are never shown.

Even if you set a date range and there are less than 100 contributors displayed for the selected period, the list still only contains contributors who are among the all-time top 100.

**Reply by guspan-tanadi** [reply]
in other words, the date range are only shown contributor that are within main [list](https://github.com/envoyproxy/envoy/graphs/contributors).

**Comment by Winbobob** [comment]
Thank you both for the help! Really appreciate it!
